server/server.go

```diff
@@ -606,9 +608,14 @@ func New(config *Config) (*Server, error) {
 	s.restore()
 	s.cleanupSandboxesOnShutdown()
 
+	bindAddress, err := knet.ChooseBindAddress(net.IP{0, 0, 0, 0})
+	if err != nil {
+		return nil, err
```

Comment:
Would the err supply enough info to the user to figure out the issue, or should we return a fmt.ErrorF() message like we do on line 622?